tests/feature/test_dedode.py

```diff
@@ -1,17 +1,19 @@
-import sys
-
 import pytest
 import torch
 
 from kornia.feature.dedode import DeDoDe
 from kornia.utils._compat import torch_version_le
 
 
+@pytest.mark.skipif(torch_version_le(2, 0, 0), reason="Autocast not supported")
 class TestDeDoDe:
-    @pytest.mark.skipif(torch_version_le(2, 0, 0), reason="Autocast not supported")
-    def test_smoke(self, dtype, device):
-        # only testing "B" as dinov2 is quite heavy
-        dedode = DeDoDe(descriptor_model="B").to(device, dtype)
+    @pytest.mark.slow
+    @pytest.mark.parametrize("descriptor_model", ["B", "G"])
+    @pytest.mark.parametrize("detector_model", ["L"])
+    def test_smoke(self, dtype, device, descriptor_model, detector_model):
+        if "G" in descriptor_model and device.type != "cuda":
+            pytest.skip('G descriptors do not support no cuda device. "LayerNormKernelImpl" not implemented for `Half`')
```

Comment:
most of the missing lines for the coverage are from dinov2, which just run on `G` descriptor.